server/views/macros/adjudicationFilter.njk

```diff
@@ -2,10 +2,21 @@
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
 
 {% macro adjudicationFilter(filter, statuses, errors, clearUrl, csrfToken) %}
+  {% set statusInRequest = filter.status.toString() %}
+  {% set statusInRequestArr = statusInRequest.split(',')%}
```

Comment:
I think these two lines of logic would be better done in the controller file (so you know exactly what you're passing as the filter). This will keep the template cleaner too, it's much easier to debug when it's on the server side

Reply:
Sorry I'm really not a fan of theses two lines of logic in the njk file - I also don't see why it's necessary as aren't you passing them through as the enum, so they'd be represented as strings in the array anyway? 
Can you not just do the `filter.status.includes(status.value)`?

Reply:
yub will do 👍 -)